vm/systemSmartContracts/auction.go

```diff
@@ -1435,6 +1435,11 @@ func (s *stakingAuctionSC) getBlsKeysStatus(args *vmcommon.ContractCallInput) vm
 	return vmcommon.Ok
 }
 
+// SetNewGasCosts is called whenever a gas cost was changed
+func (s *stakingAuctionSC) SetNewGasCosts(gasCost vm.GasCost) {
+	s.gasCost = gasCost
```

Comment:
We need to protect `s.gasCost` with a mutex

Reply:
no need. there is no possibility to call the from different go routines

Reply:
protected all the way